README.md

```diff
@@ -40,13 +40,40 @@ Then make the following edits to your serverless.yaml file:
 plugins:
   - serverless-vpc-discovery
 
+# required config
```

Comment:
The `custom` config is also provider if users choose to add config at the function level instead

Reply:
I haven't got what you mean.

And I haven't implement a logic for using `custom. vpcDiscovery` or 'functions.[name].vpcDiscovery'
It's required to have `custom.vpcDiscovery` and 'functions.[name].vpcDiscovery' as optional